e2e/old-arch/panels-suite/geomap-map-controls.spec.ts

```diff
@@ -1,58 +1,71 @@
 import { e2e } from '../utils';
 const DASHBOARD_ID = 'P2jR04WVk';
+const TIMEOUT = 60000;
```

Comment:
60s seems an eternity when the default is 4s 😕

Reply:
I think the default retry timeout for selectors is 30s, I can make it 45s if that is ok?

Reply:
Maybe for these flakier tests we should try using [Test suite retries](https://docs.cypress.io/app/guides/test-retries#Test-Suites) instead? WDYT @ashharrison90 ?

Reply:
I removed the timeout and added test suite retries instead.

Reply:
ah didn't realise it was set elsewhere 👍